Sources_v3/Controllers/ChannelListController.swift

```diff
@@ -71,6 +71,9 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
         return observer
     }()
     
+    /// Used for observing new channels creation and updating required `ChannelListQueries`.
+    private lazy var newChannelObserver = NewChannelObserver<ExtraData>(channelListQueryUpdated: worker)
+    
```

Comment:
Should be a background worker in `Client`

Reply:
What do u think about `NewChannelQueryUpdater`?